src/rascal/representations/calculator_spherical_expansion.hh

```diff
@@ -1764,35 +1776,43 @@ namespace rascal {
         // (the periodic images move with the center, so their contribution to
         // the center gradient is zero)
         if (this->compute_gradients and (atom_j_tag != atom_i_tag)) {  // NOLINT
+          // \grad_i c^j
           auto & coefficients_neigh_gradient =
               expansions_coefficients_gradient[neigh];
 
           auto && neighbour_derivative =
               radial_integral->compute_neighbour_derivative(dist, neigh);
           double df_c{cutoff_function->df_c(dist)};
           // The gradients only contribute to the type of the neighbour
-          // (the atom that's moving)
```

Comment:
Perhaps rephrase comment -- in any case, the gradients only contribute to the type of atom that's moving (i.e. w.r.t. which the derivative is being taken), right? This mixed-type notation is confusing me, but I'm assuming the multi-species tests pass...?

Reply:
it passes the tests yes.

Reply:
Perhaps writing out the sum would make it less confusing, something like `grad_i c^{ib} = sum_j (-1)^l grad_j c^{jb}`?